docs/guide/extending_online_editor.md

```diff
@@ -94,6 +100,73 @@ ezrichtext.custom_tags.ezyoutube.attributes.width.label: Width
 ezrichtext.custom_tags.ezyoutube.attributes.align.label: 'Align'
 ```
 
+### FactBox tag
+
+FactBox tag is a good example for showcasing possibilities of ezcontent property.
+Each custom tag has an ezcontent property, that contains the tag's main content.
+This property is editable by a textarea that is part of a custom tag editing tooltip. 
+
+Create the `custom_tags.yml` configuration file that will be added to `app/Resources/config`. This is sample configuration for FactBox tag:
+
+```yaml hl_lines="10"
+ezpublish:
+    system:
+        admin_group:
+            fieldtypes:
+                ezrichtext:
+                    custom_tags: [ezfactbox]
+
+    ezrichtext:
+        custom_tags:
+            ezfactbox:
+                template: '@ezdesign/fields/ezrichtext/custom_tags/ezfactbox.html.twig'
+                icon: '/assets/images/custom_tags/admin/icons/factbox.svg#factbox'
+                attributes:
+                    name:
+                        type: 'string'
+                        required: true
+                    style:
+                        type: 'choice'
+                        required: true
+                        default_value: 'light'
+                        choices: ['light', 'dark']
+```
+
+Remember to provide your own files for the template and the icon.
+Line 10 points to `ezfactbox.html.twig` template described below.
+Attributes listed below custom tag can be set when adding the tag to a RichText Field.
+
+The configuration requires an `ezfactbox.html.twig` template for the custom tag that will be placed in `/Resources/views/field_type/ezrichtext/custom_tag`:
```

Comment:
The path `/Resources/views/field_type/ezrichtext/custom_tag` seems to not be the same as the one provided in the configuration.